akirachix/Upile-Frontend
# adds lostlocate boilerplate

**What this PR does?**

    **What is this?**
    This PR introduces the foundational boilerplate for a new Next.js project. It sets up the initial structure and configuration needed for the project to begin development.

    **Why this?**
    To ensure that the project starts with a solid foundation, following Next.js best practices and including standard configurations (e.g., ESLint, Prettier, etc.) that streamline future development.

    **How are you doing this?**

-     Initialized a Next.js project using create-next-app.
-     Added folder structure and essential configurations.
-     Set up TypeScript, ESLint, and Prettier for code quality and formatting consistency.
-     Configured basic routing and placeholders for future components.

**List of updates:**

-     Created pages, components, styles directories.
-     Configured TypeScript.
-     Integrated ESLint and Prettier.
-     Added basic routing setup.
- 

**Type of change**


**How can it be tested?**

-     Clone the repository.
-     Run npm install to install dependencies.
-     Start the development server with npm run dev.
-     Verify the Next.js boilerplate structure and routing works as expected.
-
![Screenshot from 2024-09-16 10-50-07](https://github.com/user-attachments/assets/11693602-c8c7-4b8a-b2a0-ff6836938c65)


**Where is it documented?**

https://wambuinatalygakenye.atlassian.net/browse/UP-137?atlOrigin=eyJpIjoiNWZmYTdhYzQ4ZmZiNDEzODgxNDg1NTFkMDk1NTBiYjciLCJwIjoiaiJ9
